external-scripts.json

```diff
@@ -2,7 +2,6 @@
   "hubot-diagnostics",
   "hubot-help",
   "hubot-heroku-keepalive",
-  "hubot-google-images",
```

Comment:
I was unsure how the alias '!' worked and I couldn't test it (afaik) because it is for slack?

I'm assuming wozimage == !image, and woz image == ! image.

"wozimage test" works and returns me a image.  Not sure how the alias could have broken....